### PR TITLE
Restructured the directory structure to be clearer and more meaningful

### PR DESCRIPTION
**Pull request by jcanseco**

### CODE REVIEW

**Comment by joelagnel** [comment]
Could you please call the scripts as "scripts/setup-scripts". The original
structure had the word setup in it. Could you please change 'bcc' to
bcc-deps so it's clear that these are dependencies pulled externally. Also
please include a README in bcc-deps which clearly mentions which bcc commit
do these files come from. In the README, mention that these files should
not to be modified directly but rather pulled from BCC project. And that a
script is in the works to pull it. Basically it should be clear what the
purpose it. Lastly I think bcc-deps shouldn't share the src/ tree structure
with bpfd since we are also going to be syncing bpfd into bcc right ? Sharing the same tree might complicate that. So I
suggest a structure like:
bcc-deps/
src/
scripts/setup-scripts/
Its also just cleaner this way.

**Comment by jcanseco** [comment]
I made the requested changes. Please take a look. I'll let you review it first before I propagate these changes into the other PRs.

**Comment by joelagnel** [comment]
Ok. At the moment I am out of time since I am traveling. I will try to get to it tomorrow and review the new changes.

**Comment by joelagnel** [comment]
You added more commits ontop of old commits. Make sure the PR has only commits that you want merged. Use git rebase. Right now the PR shows 6 commits which seems wrong.

**Comment by jcanseco** [comment]
Okay. I cleaned up the branch using some rebase magic.

I'll also do it this way for requested changes in future PRs whenever appropriate.

File: bcc-deps/README [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+The files in 'bcc-deps/' are source files from the BCC repository that
+BPFd makes use of.
+
+They are not to be modified, and the only changes allowed to be made on these
+files are when they are updated by their newer copies from the BCC repository.
+
+These files as they are currently were pulled from the following fork of the
+BCC repository at the following commit:
+
+Repo: https://github.com/joelagnel/bcc/
```
Comment (joelagnel) [inline bcc-deps/README:13]: I guess this should be changed to point to iovisor/bcc at some point. For now its a good place holder so I'm fine with this.

**Review by joelagnel — APPROVED** [review]

**Comment by joelagnel** [comment]
For future, lets make sure commit messages are formatted properly. In one of the commits, the title and body are split. Also lets make them more descriptive. Someone reading the history should be clear what the changes do without having to refer to the PR description in github

**Pull request merged**